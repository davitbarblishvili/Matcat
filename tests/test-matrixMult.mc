func main() int {
    matrix a; 
    matrix b; 
    matrix c; 

    a = [[1,2,3],[4,5,6],[7,8,9]];
    b = [[10,11,12],[13,14,15],[16,17,18]];

    c = a * b;
    printm(c);

    return 0;
}