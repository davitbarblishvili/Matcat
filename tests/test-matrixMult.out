(
[84 90 96 ]
[201 216 231 ]
[318 342 366 ]
)